.enb/lib/page.js

```diff
@@ -0,0 +1,126 @@
+var techs = require('../techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl],
+        [techs.bem.depsOld],
+        [techs.bem.files]
+    ]);
+
+    // css
+    node.addTechs([
+        [techs.css.stylusWithAutoprefixer, { browsers : browsers }],
+        [techs.css.stylus, { target : '?.ie.css', sourceSuffixes : ['styl', 'ie.styl'] }],
+    ]);
+    node.mode('development', function() {
```

Comment:
If you depend on the environment variable, you may have problems with parallel assembly via JS API and option `mode` will not work.